File: JavaScriptPractice/Leetcode414.js
var thirdMax = function(nums) {
    let theSet = new Set(nums);
    let theSetArray = Array.from(theSet);
    theSetArray.sort((a, b) => a - b)
    if (theSetArray.length<3) {
        return theSetArray[theSetArray.length-1];
        
    }
    else 
    {
        return theSetArray[theSetArray.length-3];
    }

};

File: CSharpPractice/LeetcodePractice/Leetcode414.cs
public class Solution414 {
    public int ThirdMax(int[] nums) {
        HashSet<int> theSet = new HashSet<int>(nums);
        List<int> theSetList = new List<int>(theSet);
        theSetList.Sort();
        if (theSetList.Count<3)
        {
            return theSetList[theSetList.Count-1];
        }
        else
        {
            return theSetList[theSetList.Count-3];
        }
        
        
    }
}